migrations/1702134678728_pox_4_events.js

```diff
@@ -171,6 +171,8 @@ exports.up = pgm => {
       WHEN 'stack-aggregation-increase' THEN
         reward_cycle IS NOT NULL AND
         amount_ustx IS NOT NULL
+      WHEN 'revoke-delegate-stx' THEN
+        delegate_to IS NOT NULL
```

Comment:
Can you make this change in a new migration file? Otherwise it's a breaking change and requires a major version bump (and costly upgrade)

Reply:
Ah yes, wanted to ask about this, if it's live somewhere. Will do 👍🏻